packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -279,6 +283,18 @@ class ViewSettingsDialog extends UI5Element {
 		return "";
 	}
 
+	get shouldBuildSort() {
+		return this.sortItems.length;
```

Comment:
I would make sure that a getter like that would return a <code>Boolean</code>, in order to avoid some false assumptions when used.